src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -498,9 +500,44 @@ private void ProcessRenderQueue()
             {
                 ProcessRenderQueue();
             }
+
+            if (asyncDisposalTasks != null)
+            {
+                _ = ProcessAsyncDisposables(asyncDisposalTasks);
+            }
+
             ComponentsProfiling.Instance.End();
         }
 
+        private async Task ProcessAsyncDisposables(List<Task> asyncDisposalTasks)
+        {
+            await Task.WhenAll(asyncDisposalTasks);
+            List<Exception> exceptions = null;
+
+            int count = asyncDisposalTasks.Count;
+            for (var i = 0; i < count; i++)
```

Comment:
This is intented to filter out cancellation exceptions? Perhaps comment why

Reply:
this is gone now